### PR TITLE
Some Italian chain stores

### PR DESCRIPTION
**Pull request by naposm**
Hi! These are some Italian chain stores. I'll update and add more of them next monday when some shops will open again!

### CODE REVIEW

**Comment by PanierAvide** [comment]
File seem to be messed up, maybe an encoding issue ? Please check and fix your edits.

**Comment by naposm** [comment]
@PanierAvide Is it correct now?

**Comment by naposm** [comment]
@PanierAvide Yes, sorry. I manually changed it and github seems to parse it correctly now 😄

**Comment by PanierAvide** [comment]
Looks better regarding encoding issue. Just one last step : lines are sorted by country, category, brand name, can you make sure to move Italy lines in the right place in file ? Thank you :wink:

**Comment by naposm** [comment]
@PanierAvide done 😄

File: rules.csv [hunk 1]
```diff
@@ -1121,3 +1130,4 @@ US,supermarket,Fred Meyer,Q5495932,open_adapted,07:00-21:00,,Dedicated to senior
 US,supermarket,Safeway,Q1508234,open_adapted,07:00-22:00,,"Shops open, offers « contact free » delivery and pickup at stores",https://www.safeway.com/faq/covid-19-faq.html,https://local.safeway.com/search.html
 US,supermarket,Walgreens,Q1591889,open_adapted,09:00-17:00,,,https://news.walgreens.com/our-stories/covid-19-faq.htm,
 US,supermarket,Walmart,Q483551,open_adapted,07:00-20:30,,Dedicated to seniors shoppers between 07:00 and 08:00,https://corporate.walmart.com/here-for-you,https://corporate.walmart.com/important-store-info
+
```
Comment (PanierAvide) [inline rules.csv]:
```suggestion
```

**Comment by PanierAvide** [comment]
Thanks :+1:

**Pull request merged**